examples/layer_norm.py

```diff
@@ -0,0 +1,64 @@
+from __future__ import annotations
+
+from typing import Any
+
+import torch
+
+import helion
+from helion._testing import run_example
+import helion.language as hl
+
+
+@helion.kernel(static_shapes=True, use_default_config=True)
```

Comment:
We probably want to remove `use_default_config=True` because it prevents autotuning. For local development, we can set `HELION_USE_DEFAULT_CONFIG=1` env var.